mgmt_cli_test.py

```diff
@@ -1006,7 +1006,6 @@ def _suspend_and_resume_task_template(self, task_type):
         assert suspendable_task.wait_for_status(list_status=[TaskStatus.RUNNING], timeout=300, step=5), \
             f"task {suspendable_task.id} failed to reach status {TaskStatus.RUNNING}"
         with mgr_cluster.suspend_manager_then_resume(start_tasks=False):
-            mgr_cluster.suspend()
             assert suspendable_task.wait_for_status(list_status=[TaskStatus.STOPPED], timeout=300, step=10), \
                 f"task {suspendable_task.id} failed to reach status {TaskStatus.STOPPED}"
             mgr_cluster.resume(start_tasks=True)
```

Comment:
is this resume needed ? since the context manager here would resume, isn't it ?

Reply:
and the assert below needs to go out of the context, isn't it ?

Reply:
Correct, done.